python/seldon_core/microservice.py

```diff
@@ -361,6 +387,13 @@ def grpc_prediction_server():
     else:
         server1_func = None
 
+    def rest_metrics_server():
+        app = seldon_microservice.get_metrics_microservice(seldon_metrics)
+        app.run(host="0.0.0.0", port=metrics_port)
+
+    logger.info("REST metrics microservice running on port %i", metrics_port)
```

Comment:
This should be part of the function above

Reply:
Added like that only because `logger.info` for `rest_prediction_server` and `grpc_prediction_server` are also defined outside and I thought there was some logic for it.